tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -175,29 +193,53 @@ export interface uploadResourceArgs {
   page: Page
   resources: File[]
   to?: string
-  createVersion?: boolean
+  option?: string
 }
 
 export const uploadResource = async (args: uploadResourceArgs): Promise<void> => {
-  const { page, resources, to, createVersion } = args
+  const { page, resources, to, option } = args
   if (to) {
     await clickResource({ page: page, path: to })
   }
 
   await page.locator(resourceUploadButton).click()
   await page.locator(fileUploadInput).setInputFiles(resources.map((file) => file.path))
 
-  if (createVersion) {
-    await page.locator(actionSecondaryConfirmationButton).click()
-    // @TODO check if upload was successful
-  }
-
-  await page.locator(uploadInfoCloseButton).click()
+  if (option) {
+    switch (option) {
+      case 'Skip': {
+        await page.locator(actionCancelButton).click()
+        break
+      }
+      case 'Replace': {
+        await page.locator(actionSecondaryConfirmationButton).click()
+        await page.locator(uploadInfoCloseButton).click()
+
+        await waitForResources({
+          page: page,
+          names: resources.map((file) => path.basename(file.name))
+        })
+        break
+      }
+      case 'keep both': {
+        await page.locator(actionConfirmationButton).click()
```

Comment:
```suggestion
        await page.locator(actionKeepBothButton).click()
```

Reply:
this button is used in other places too for creating files as a confirmation button so using the `actionKeepBothButton` name may be not suitable. @saw-jan